contribs/gmf/src/directives/themeselector.js

```diff
@@ -18,6 +18,7 @@ gmf.themeselectorDirective = function() {
     restrict: 'E',
     controller: 'gmfThemeselectorController',
     scope: {
+      'defaultTheme': '@gmfThemeselectorDefaulttheme',
```

Comment:
The `@htmlAttribute` should be added, see: https://github.com/camptocamp/ngeo/blob/master/contribs/gmf/src/directives/search.js#L47-L50